File: programmers/Level01/Programmers_12932.java
package programmers.Level01;

public class Programmers_12932 {
    public int[] solution(long n) {
        String s = Long.toString(n);
        
        int[] answer = new int[s.length()];

        for(int i = 0; i<answer.length; i++){
            answer[i] = s.charAt(s.length()-1-i);
        }

        return answer;
    }    
}

File: programmers/Swift/Level01/Programmers_12932.swift
final class Programmers_12932 { 
    func solution(_ n:Int64) -> [Int] {
        var num = Int(n)
        var arr : [Int] = []
        while num > 0 {
            arr.append(num % 10)
            num /= 10
        }
        return arr
    }
}